CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 Friendly reminder, we have a [bug bounty program](https://hackerone.com/cosmos).
 
+## Unreleased
+
+### Improvements
+
+- [logging] [\#9191](https://github.com/tendermint/tendermint/issues/9191) Use go 1.19 for additional safety: https://tip.golang.org/doc/go1.19
```

Comment:
I'm not sure I'd include this in a change log, because we're not changing the requirement for downstream users of tendermint to use go1.19, just changing the way that we build and test tendermint. 

I think we'd need to be the last in the ecosystem to move to go1.19 (e.g. after apps/the sdk), otherwise we could complicate downstream users' builds and release timelines by forcing their hand (which I think is why pull requests like this are good, that update what we build with but not what we require.)

Reply:
happy to remove